### DISCUSSION TITLE
Open URL link in main browser post response (how to?)

### DISCUSSION THREAD

**Discussion by Razorator** [usebruno/bruno#2333] [Q&A]
I have a login request that I get a JWT token from and I want my browser to automatically open the url with the jwt query post response.

My current workaround is keeping the Chrome console open, console.log the url+jwt, and then copy-paste into my browser (clicking it doesn't work). EDIT: Is it possible to have the console auto open with bruno? And can the environment preselect to the last one used?

It's not a big issue, but I like optimizing my workflow. :)

**Comment by tkozlovsage** [comment]
const { spawn } = require("child_process");
const url = [your url here]
if (process.platform === "win32")
{ 
   spawn("rundll32.exe", ["url.dll,FileProtocolHandler", url], {
      detached: true, stdio: "ignore"
    }).unref();
}
